ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/websocket/WebSocketService.java

```diff
@@ -103,6 +105,12 @@ public WebSocketService(
     };
   }
 
+  private Handler<HttpServerRequest> httpHandler() {
+    System.out.println("HTTP Handler");
```

Comment:
S.o.p line, why not use a Logger?

Reply:
Nah, I forgot to delete this line :)
I'm impressed our static code checked doesn't complain about System.out.println statements on the code.